lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -60,13 +61,24 @@ class DSLocationMessageBubble extends StatelessWidget {
           mainAxisSize: MainAxisSize.min,
           crossAxisAlignment: CrossAxisAlignment.center,
           children: [
-            DSCachedNetworkImageView(
-              url:
-                  'https://maps.googleapis.com/maps/api/staticmap?&size=360x360&markers=$latitude,$longitude&key=${DSAuthService.googleKey}',
-              placeholder: (_, __) => _buildLoading(),
-              align: align,
-              style: style,
-            ),
+            double.tryParse(latitude) != null &&
+                    double.tryParse(longitude) != null
+                ? DSCachedNetworkImageView(
+                    url:
+                        'https://maps.googleapis.com/maps/api/staticmap?&size=360x360&markers=$latitude,$longitude&key=${DSAuthService.googleKey}',
+                    placeholder: (_, __) => _buildLoading(),
+                    align: align,
+                    style: style,
+                  )
+                : const SizedBox(
+                    width: 240,
+                    height: 240,
+                    child: Icon(
+                      DSIcons.file_image_broken_outline,
+                      size: 80,
+                      color: DSColors.neutralMediumCloud,
```

Comment:
This color should depend on the bubble style, similar on how it's done by the ```DSCachedNetworkImageView._defaultErrorWidget``` method.

```suggestion
                      color: style.isLightBubbleBackground(align) ? DSColors.neutralMediumElephant : DSColors.neutralMediumCloud,
```

Reply:
<img width="500" alt="image" src="https://github.com/takenet/blip-ds-flutter/assets/91232521/0996da10-cdfb-47ca-b536-2828d49b1973">